fuel-vm/src/interpreter/executors/main.rs

```diff
@@ -42,52 +44,113 @@ impl<T> Interpreter<PredicateStorage, T> {
     /// This is not a valid entrypoint for debug calls. It will only return a `bool`, and not the
     /// VM state required to trace the execution steps.
     pub fn check_predicates<Tx>(
-        checked: Checked<Tx>,
+        checked: &Checked<Tx>,
```

Comment:
It's usually best practice to take an owned value when it get's cloned below as it tells the user of this api that the value is used.

Reply:
fixed, reverted back to format before a mutable reference was used to both check and estimate predicates via a boolean passed into this function that was removed